.eslintrc.js

```diff
@@ -1,35 +1,38 @@
-const jestRules = require('eslint-plugin-jest').rules;
-const jestDomRules = require('eslint-plugin-jest-dom').rules;
-const testLibRules = require('eslint-plugin-testing-library').rules;
-
-const disableAllRules = (pluginName, rules) => {
-	return Object.keys(rules).reduce((ac, ruleName) => {
-		return {
-			...ac,
-			[`${pluginName}/${ruleName}`]: 'off',
-		};
-	}, {});
-};
-
-// We need to turn off jest rules for cypress because
-// Cypress uses expect.
-const cypressRuleDisables = {
-	...disableAllRules('jest', jestRules),
-	...disableAllRules('jest-dom', jestDomRules),
-	...disableAllRules('testing-library', testLibRules),
-};
-
 module.exports = {
-	extends: '@nciocpl/eslint-config-react',
+	env: {
```

Comment:
This should still import from our lint configs, overridding whatever you may need to. A couple of things:
1. Update to 1.0.0 for the package
2. Look at https://github.com/NCIOCPL/cgov-standards-xt/blob/v1.0.0/packages/eslint-config-react/minimal.js. It does not force the parser options and whatnot -- if that is the issue. If you need some version bumped or not, shout -- there are ways to force package.json to download the dependencies you want.

Reply:
TL;DR; I have concerns with ending up in the place we were WITHOUT the eslint-config-react, which was each React app had different rules and upgrading those rules became "hard" because things got missed each time.